docs/source/author/content_style_guide.rst

```diff
@@ -0,0 +1,231 @@
+Content Style Guide
+##############################################################################
+
+.. note::
+
+  This page is currently intended for internal staff only.
```

Comment:
Possibly add a version for all contributors will be added once the project is ready for outsider contributions.